go.mod

```diff
@@ -14,10 +14,9 @@ require (
 	github.com/Azure/go-autorest/autorest/to v0.2.0
 	github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190822182118-27a4ced34534
 	github.com/JeffAshton/win_pdh v0.0.0-20161109143554-76bb4ee9f0ab
-	github.com/Microsoft/go-winio v0.4.14
-	github.com/Microsoft/hcsshim v0.0.0-20190417211021-672e52e9209d
+	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
```

Comment:
is there a tagged level we can use?

Reply:
asked them the question in https://github.com/microsoft/go-winio/issues/167